frame/base-fee/src/lib.rs

```diff
@@ -239,6 +242,21 @@ pub mod pallet {
 	}
 }
 
+impl<T: Config> Pallet<T> {
+	pub fn set_base_fee_per_gas_inner(value: U256) -> Weight {
```

Comment:
Can you rename the name to without `inner`? I believe this won't conflict with `Call` as they are stored in a separate enum.

Reply:
Yeah I also believed that was the case and previously tried, but the compiler throws `duplicate definitions` error.